Cargo.toml

```diff
@@ -16,17 +16,27 @@ features = ["docs"]
 rustdoc-args = ["--cfg", "feature=\"docs\""]
 
 [features]
-default = ["async_std", "cookie-secure"]
+default = ["cookie-secure"]
 docs = ["unstable"]
 unstable = []
 hyperium_http = ["http"]
-async_std = [] # "async-std" when it is not default
+async_std = ["async-std"] # "async-std" when it is not default
 cookie-secure = ["cookie/secure"]
 
+[dependencies.futures-io]
```

Comment:
Instead of using `futures` and `piper` I would suggest using `async-std` with the default features, especially the runtime turned off. This way nothing changes in terms of code, but you are not forced to pull in the async-std runtime anymore, just the tooling like channels etc.

Reply:
with https://github.com/async-rs/async-std/pull/806 this

```toml
async-std = { version = "1.6", default-features = false, features = ["unstable"] }
```